src/generate/mathjax.ts

```diff
@@ -11,13 +11,20 @@ import path from 'path'
 
 const dest = path.join(__dirname, '..', 'shared', 'styles', 'mathjax.css')
 
+interface Result {
+  css?: string
+  errors?: string[]
+}
+
 MathJax.typeset(
   {
     css: true
   },
-  (result: any) => {
+  (result: Result) => {
     const { errors, css } = result
-    if (errors) errors.map(console.error)
+    if (typeof errors !== 'undefined' && typeof errors.length !== 'undefined') {
```

Comment:
Unwieldy, using due to `strict-boolean-expressions` rule; adding `ignore-rhs` might make things like this a bit less clunky? Not added in this PR as the first step is to get the reporting working with the current rule set.

Reply:
Can it be simplified to `errors?.map(err => console.error(err))`

We had ongoing discussions about that rule e.g. https://github.com/stencila/encoda/issues/199 & https://github.com/stencila/dev-config/issues/4.

Reply:
Yeah, I checked, it can.

Reply:
But,...that creates a different warning, so : `if (errors !== undefined) errors.map(err => console.error(err))`

Reply:
Implemented in d35f125.